### PR TITLE
net|rpc: getpeerinfo - Add verified_pubkey_hash, fix a typo in verified_proregtx_hash docs

### PR DESCRIPTION
**Pull request by xdustinface**

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
Needs rebase

**Comment by PastaPastaPasta** [comment]
Needs rebase

**Review by UdjinM6 — DISMISSED** [review]
utACK

File: src/rpc/net.cpp [hunk 1]
```diff
@@ -83,6 +83,9 @@ UniValue getpeerinfo(const JSONRPCRequest& request)
             "    \"verified_proregtx_hash\": h, (hex) Only present when the peer is a masternode and succesfully\n"
             "                               autheticated via MNAUTH. In this case, this field contains the\n"
             "                               protx hash of the masternode\n"
+            "    \"verified_pubkey_hash\":   h, (hex) Only present when the peer is a masternode and succesfully\n"
+            "                               autheticated via MNAUTH. In this case, this field contains the\n"
```
Comment (thephez) [inline src/rpc/net.cpp:87]:
```suggestion
            "                               authenticated via MNAUTH. In this case, this field contains the\n"
```

File: src/rpc/net.cpp [hunk 2]
```diff
@@ -83,6 +83,9 @@ UniValue getpeerinfo(const JSONRPCRequest& request)
...
+            "                               hash of operator public key of the masternode\n"
```
Comment (thephez) [inline src/rpc/net.cpp:88]:
```suggestion
            "                               hash of the masternode's operator public key\n"
```

**Review by thephez — COMMENTED** [review]
A typo and minor wording change

**Review by thephez — APPROVED** [review]
utACK

File: src/rpc/net.cpp [hunk 3]
```diff
@@ -84,8 +84,8 @@ UniValue getpeerinfo(const JSONRPCRequest& request)
             "                               autheticated via MNAUTH. In this case, this field contains the\n"
             "                               protx hash of the masternode\n"
             "    \"verified_pubkey_hash\":   h, (hex) Only present when the peer is a masternode and succesfully\n"
-            "                               autheticated via MNAUTH. In this case, this field contains the\n"
-            "                               hash of operator public key of the masternode\n"
+            "                               authenticated via MNAUTH. In this case, this field contains the\n"
```
Comment (UdjinM6) [inline src/rpc/net.cpp:87]: same typo above 🙈 but it's out of the scope for this PR 😄

**Review by UdjinM6 — APPROVED** [review]
utACK

File: src/rpc/net.cpp (hunk 3 above)
Comment (xdustinface) [reply src/rpc/net.cpp:87]: Don't you mean c09b83a? :D

Comment (UdjinM6) [reply src/rpc/net.cpp:87]: oh 🙈

**Review by PastaPastaPasta — APPROVED** [review]
no tests? terrible!

utACK

**Comment by xdustinface** [comment]
> no tests?

See #3930 (45a15fc1262236372245ebf8a16b1e467632616c) :D

**Pull request merged**